.nix/melee.nix

```diff
@@ -40,6 +41,7 @@ stdenv.mkDerivation {
 
   nativeBuildInputs = [
     decomp-toolkit
+    devkitppc
```

Comment:
I was trying to factor this out, do we need it for something still?

Reply:
It's not needed but it's nice for development